internal/commands/push.go

```diff
@@ -34,9 +34,10 @@ type pushOptions struct {
 func pushCmd(dockerCli command.Cli) *cobra.Command {
 	var opts pushOptions
 	cmd := &cobra.Command{
-		Use:   "push [<app-name>]",
-		Short: "Push the application to a registry",
-		Args:  cli.RequiresMaxArgs(1),
+		Use:     "push [APP_NAME] --tag TARGET_REFERENCE [OPTIONS]",
```

Comment:
APP_NAME shouldn't have brackets.

Reply:
[--tag TARGET_REFERENCE]

Reply:
It's optional as we'll pick up the `.dockerapp` file in the current directory if nothing is specified.

Reply:
Added an issue about this 👍